fury/utils.py

```diff
@@ -138,6 +138,40 @@ def numpy_to_vtk_cells(data, is_coords=True):
     return cell_array
 
 
+def vtk_vertices_from_actor(actor):
+    """Access to vtk vertices from actor.
+
+    Parameters
+    ----------
+    actor : actor
+
+    Returns
+    -------
+    vertices : vtkarray
+
+    """
+    return actor.GetMapper().GetInput().GetPoints().GetData()
```

Comment:
to remove, this function already exists in the same file.

Look at [line 1011](https://github.com/fury-gl/fury/blob/master/fury/utils.py#L1011)

if you just want the vtk type, you can add an option in this function.

Reply:
Fixed